R/log_import.R

```diff
@@ -0,0 +1,80 @@
+#' Write the log data to the database.
+#'
+#' @param con Connection to the database to be written to.
+#' @param log_table The name of the table to write to.
+#' @param log_data The log data to be written to the DB.
+#'
+#' @keywords internal
+write_log <- function(con, log_table, log_data) {
+  DBI::dbAppendTable(con, log_table, log_data)
+}
+
+#' Get the data to be written to the import log
+#'
+#' @param import_path Path to the import process directory.
+#' @param comment An optional comment for this import run.
+#'
+#' @return The prepared data for the log table. This is the name
+#' of the import, the date, comment and git information including
+#' user name, user email, current branch and hash of HEAD.
+#'
+#' @keywords internal
+get_log_data <- function(import_path, comment) {
```

Comment:
`build_log_data` perhaps? This name confused me because it's not getting anything as such

Reply:
Yep that's definitely more accurate for what it is doing. I just can't resist writing getters :)

I've made that change.